ion_phys/tests/test_tls.py

```diff
@@ -0,0 +1,81 @@
+""" Simple rate equations example of a 2 level system"""
+
+import unittest
+
+from ion_phys.ions.ca43 import Ca43, ground_level, P32
+from ion_phys import Laser
+from ion_phys.rate_equations import Rates
+
+
+def steady_state(intensity):
+    "Steady state poulation in the P-state for resonant intensity /I0"
+    return intensity / (2 * intensity + 1)
+
+
+class TestTLS(unittest.TestCase):
```

Comment:
The way you've set this up, it's really a two-state system, not two *levels* (e.g. you're not worrying about degeneracies). That's fine, but we should change the docstrings/names to match.